src/main/java/org/example/TicTacToe.java

```diff
@@ -0,0 +1,97 @@
+package org.example;
+
+import org.example.boardUpdating.CreatingAnArrayOfSpaces;
+import org.example.boardUpdating.WrittingToAnArray;
+import org.example.arraysCreating.Paair;
+import org.example.factoring.ReadingTheGame;
+import org.example.methoding.Comment;
+import org.example.methoding.Methods;
+
+import java.util.*;
+
+public class TicTacToe {
+    public static void main(String[] args) {
+
+        //-----------------------deklaracja zmiennych --------------------------------------------------------
+        List<Paair> listPaair = new ArrayList<>();
+        List<Integer> listNumbers = new ArrayList<>();
+        Random random = new Random();
+        String pl, pl1, pl2, turn, option, save, select;
+        char[][] board = CreatingAnArrayOfSpaces.AnArrayOfSpaces(new char[3][3]);
+        int fieldNumber, state;
+        //-------------------wybór opcji wczytaj stan gry / rozpocznij nową grę ------------------------------
+       option = Methods.SelectAnOption(Comment.GameStateOptions());
```

Comment:
Bardzo duzo metod statycznych, ciezko bedzie z testami czegos takiego :( 
ale za to czytelne :)